cluster-autoscaler/cloudprovider/clusterapi/clusterapi_converters.go

```diff
@@ -69,6 +85,22 @@ func newMachineSetFromUnstructured(u *unstructured.Unstructured) *MachineSet {
 		Status: MachineSetStatus{},
 	}
 
+	machineSpec, found, err := unstructured.NestedMap(u.Object, "spec", "template", "spec")
+	if err == nil && found {
+		machineSpecUnstructured := unstructured.Unstructured{Object: machineSpec}
+		machineSet.Spec.Template.Spec.Labels = machineSpecUnstructured.GetLabels()
+		taints, _, _ := unstructured.NestedSlice(machineSpec, "Taints")
```

Comment:
Same comment as above
```suggestion
		taints, _, _ := unstructured.NestedSlice(machineSpec, "taints")
```